platform/viewer/src/studylist/StudyListRoute.js

```diff
@@ -58,8 +58,12 @@ function StudyListRoute(props) {
   const [pageNumber, setPageNumber] = useState(0);
   const appContext = useContext(AppContext);
   // ~~ RESPONSIVE
-  const displaySize = useMedia(
-    ['(min-width: 1750px)', '(min-width: 1000px)', '(min-width: 768px)'],
+  const displaySize = useDisplayMediaSize(
```

Comment:
So `useDisplayMediaSize` is the previous implementation of `useMedia`

Reply:
At least from an API standpoint

Reply:
yes.. but I`m ok if you believe we should preserve it as it was